tests/01CreateClient.js

```diff
@@ -10,7 +14,130 @@ module.exports = {
       .waitForElementVisible('button[type="submit"]', 1000)
       .click('button[type="submit"]')
       .pause(1000)
-      .assert.containsText('.side-nav', 'Logout')
```

Comment:
These are all great tests! We need to preserve the first test `'Create Client'` as is. Add other tests called something like `'Make Student'`, `'Create Location'`, `'Create Term'`, `'Create Course'`, `'Make Payment'`, and link them like this https://github.com/AustinCodingAcademy/intermediate-workbook/blob/a47eab3e25b77b4b588532e287ff6975f00ee300/apps/07jquery-tic-tac-toe/test/test.js